chat.iamb.iamb.yml

```diff
@@ -0,0 +1,48 @@
+id: chat.iamb.iamb
+runtime: org.freedesktop.Platform//23.08
+sdk: org.freedesktop.Sdk//23.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: iamb
+rename-icon: "iamb"
+rename-desktop-file: "iamb.desktop"
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+appstream-compose: false
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    - CARGO_HOME: /run/build/iamb/cargo
+    - IAMB_XDG_NAME: chat.iamb.iamb
+modules:
+  - name: iamb
+    buildsystem: simple
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --profile release-lto --verbose
+      - install -Dm755 ./target/release-lto/iamb -t /app/bin/
+      - install -Dm644 ./iamb.desktop -t /app/share/applications/
+      - install -Dm644 ./config.example.toml -t /app/share/iamb/
+      - install -Dm644 ./docs/iamb.1 -t /app/share/man/man1/
+      - install -Dm644 ./docs/iamb.5 -t /app/share/man/man5/
+      - install -Dm644 ./docs/iamb.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 ./docs/iamb.metainfo.xml -T /app/share/metainfo/chat.iamb.iamb.metainfo.xml
+      - |
+        sed -i \
```

Comment:
don't use sed like that. it's failure mode is inexistant and it's unreadable. just use `rename-appdata-file` like you do for the .desktop file.

Reply:
Removed in df7a313.